Service/Runner.php

```diff
@@ -221,29 +224,6 @@ public function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId)
             $jobId
         );
 
-        if (($action == 'run') && ($component->getStagingStorage()['input'] != 'none')) {
-            $dataLoader = new DataLoader(
```

Comment:
hm, škoda, že to nechytil nějakej automat

Reply:
haha, no jo :) problem bylo to volani setFeatures(), timpadem byl ten kod used, jinak by to odchytil codeclimate - ten bych mozna zkusil vyhledove zase nasetupovat, uz se dost zlepsil, takze nevybleje 10000 erroru a da se rozumneji filtrovat